src/libs/AxelarQueryAPI.ts

```diff
@@ -231,9 +255,20 @@ export class AxelarQueryAPI {
             .div(10000)
             .add(baseFee)
         : destTxFee.add(baseFee);
-    if (isGMPExpressTransaction) fee = fee.add(expressFee);
+    if (gmpExpressTransactionDetails?.isGMPExpressTransaction) fee = fee.add(expressFee);
 
-    return fee.toString();
+    return getFullFeeBreakdown
+      ? {
+          baseFee,
+          expressFee,
+          dynamicFee: destTxFee.toString(),
+          fee: fee.toString(),
+          gasLimit,
+          gasMultiplier,
+          srcGasPrice: srcGasPrice.toString(),
+          minGasPrice: minGasPrice === "0" ? "NA" : minGasPrice,
```

Comment:
maybe the full axelarscan API response should be included to make it future proof? We just type the known params in it